Makefile

```diff
@@ -146,6 +146,7 @@ else
 endif
 	SCHEME=${SCHEME} \
 	IDRIS2_VERSION=${IDRIS2_VERSION} \
+	MAKE=${MAKE} \
```

Comment:
you could do an `export MAKE` in config.mk instead.

Reply:
Thanks.  Also removed `${SCHEME}` and `${IDRIS2_VERSION}` as they are already exported too.